pyproject.toml

```diff
@@ -440,24 +439,15 @@ deps =
     poetry
 setenv =
     DOC_PYTHON_VERSION = python3.11  # Keep this in sync with .readthedocs.yml and any CI scripts
-    # Skip pre-commit checks that are explicitly performed by tox or not needed
-    SKIP = ruff-format,docformatter,ruff,pylint,pyright,pyright-verifytypes,pyroma,poetry-audit,file-contents-sorter
+    # Skip pre-commit checks that are not needed
+    SKIP = file-contents-sorter
     PYRIGHT_PYTHON_GLOBAL_NODE = False  # This will cause python-pyright to use nodeenv to install node rather than use the system node
 commands_pre =
     poetry install --no-root --without=main
 commands =
-    !tests: safety check --ignore=70612
-    !tests: ruff format --check --diff .
-    !tests: docformatter --check --diff .
-    !tests: ruff check .
-    !tests: pylint .
-    !tests: pyright .
-    !tests: pyright --verifytypes tm_devices --ignoreexternal
```

Comment:
Should these changes be removed from here? The verbosity on passing runs is greatly reduced (verbosity on failing runs is identical).